packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1517,6 +1517,19 @@ class Positioned extends ParentDataWidget<Stack> {
        bottom = null,
        super(key: key, child: child);
 
+  /// Creates a Positioned object with [left], [top], [right], and [bottom] set
+  /// to 0.0.
+  Positioned.stretch({
+    Key key,
+    Widget child,
```

Comment:
I'd still let you provide values for left, top, right, and bottom and just have 0.0 be the default for args you don't provide.